models/DataObject/ClassDefinition/Data/Datetime.php

```diff
@@ -309,24 +311,19 @@ public function getFromWebserviceImport($value, $object = null, $params = [], $i
         $timestamp = strtotime($value);
         if (empty($value)) {
             return null;
-        } elseif ($timestamp !== false) {
+        }
+        if ($timestamp !== false) {
             return $this->getDateFromTimestamp($timestamp);
-        } else {
-            throw new \Exception('cannot get values from web service import - invalid data');
         }
+        throw new \Exception('cannot get values from web service import - invalid data');
     }
 
     /**
-     * @return Date
+     * @return int|null
      */
     public function getDefaultValue()
     {
-        if ($this->defaultValue !== null) {
-            return $this->defaultValue;
-        //return new Date($this->defaultValue);
-        } else {
-            return 0;
-        }
+        return $this->defaultValue;
```

Comment:
See comments from @weisswurstkanone:

> my interpretation is that
> 
> public $defaultValue
> 
> should be int | null.
> 
> Setter do the conversion (in case of string for example).
> 
> Getter should return $this->defaultValue as it is.

Reply:
@blankse 👍 fine for me! 😊